ambry-quota/src/main/java/com/github/ambry/quota/capacityunit/AmbryCapacityUnitQuotaEnforcer.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2021 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.quota.capacityunit;
```

Comment:
capacityunit is a strange name for a package, not very intuitive.  Is this meant to be for request/bandwidth quotas as opposed to storage?  Consider renaming.

Reply:
I do not want to name it as request because both capacity unit and storage quotas are basically quotas for user requests. Also bandwidth doesn't really convey the meaning appropriately.
My reason for using `capacityunit` as the package name is to convey `quota.capcacityunit` is a package that deals with capacityunit type of quota. While `quota.storage` is a package that deals with storage type of quota.